developer/tests/@testExamples/testExamples.m

```diff
@@ -67,6 +67,16 @@ function tutorial_1_test(testCase)
             % Verification
             % TODO: How to verify correct run, apart from no-error?
         end
+
+        %% tutorial_2_SPM_CH30
+        function tutorial_2_test(testCase)
+            % Test that tutorial_2_SPM_CH30 runs out-of-the-box
+
+            tutorial_2_SPM_CH30()
+
+            % Verification
+            % TODO: How to verify correct run, apart from no-error?

```

Comment:
Can we save the output similar to the use cases?

Reply:
Yes, that should be possible. However, that would be outside of this test function, in the .yml files. And does not give a way of automatically verifying the run is correct.

Reply:
Yes, I meant modifying the .yml file.

Without pre-generated results, it is impossible to automatically verify the output, however, it is not in the pipeline for the near future. It is also true for the use cases. So far, it requires a manual check, which I do time-to-time.

Reply:
I've added the result-data upload to the .ym file.